bundler.d/assets.rb

```diff
@@ -1,9 +1,9 @@
 group :assets do
   gem 'jquery-turbolinks', '~> 2.1'
-  gem 'jquery-ui-rails', '< 5.0.0'
-  gem 'patternfly-sass', '>= 3.32.1', '< 3.38.0'
+  gem 'jquery-ui-rails', '~> 5.0'
+  gem 'patternfly-sass', '~> 3.59'
```

Comment:
Can this get in without the sprockets update?

Reply:
yeah sure, I'll split it.